src/base/memory_monitor_cl.hpp

```diff
@@ -24,10 +24,15 @@
 #ifndef _MEMORY_MONITOR_CL_HPP_
 #define _MEMORY_MONITOR_CL_HPP_
 
+#include <algorithm>
+
 #include "memory_monitor_common.h"
 
+constexpr int DEFAULT_OPTION_PRINT_CNT = 5;
```

Comment:
I'd put it in the `memmon` function as a local variable.

Reply:
Done.